apps/docs-snippets/src/guide/predicates/send-and-spend-funds-from-predicates.test.ts

```diff
@@ -195,6 +195,7 @@ describe('Send and Spend Funds from Predicates', () => {
     const amountToReceiver = 200;
 
     // #region send-and-spend-funds-from-predicates-8
+    // Create the transaction for transferring funds from the predicate.
```

Comment:
How come changes were made to the original snippet?

Reply:
This file should be deleted - will do this in #3369